enterprise/frontend/src/metabase-enterprise/audit_app/index.js

```diff
@@ -20,14 +20,10 @@ import { isAuditDb } from "./utils";
 export function initializePlugin() {
   if (hasPremiumFeature("audit_app")) {
     // Add new menu item function
-    const menuItemFunction = (user) => [
+    const menuItemFunction = (user, isExternalPage) => [
       <Menu.Item
         component={ForwardRefLink}
-        to={
-          isInternalUser(user)
-            ? `/admin/people/${user.id}/unsubscribe`
-            : `/admin/tenants/people/${user.id}/unsubscribe`
-        }
+        to={Urls.unsubscribeUser(user, isExternalPage)}
```

Comment:
We can now unsubscribe external users from the "People" page, so the route is determined by what page they were on (People or External Users) rather than the user type.